frontend/src/utils/partnerUtils.js

```diff
@@ -74,11 +70,12 @@ export const getPartnerState = (partner) => {
 export const getPartnerStateColor = (state) => {
   if (state === PartnerStates.REQUEST.value) {
     return "yellow";
-  } else if (state === PartnerStates.INACTIVE.value) {
+  } else if (state === PartnerStates.ABANDONED.value) {
```

Comment:
@etschelp  Do you normalize this in the backend? I'm asking because Partners/Connections based on explicit QR Code invitations will still be using the RFC160 Connection Protocol since smartphone wallets have not yet implemented DID Exchange.

Reply:
Ah, inactive was something we defined anyways...